packages/starrating/src/react/star.tsx

```diff
@@ -65,38 +73,65 @@ function HalfStarIcon(props) {
   )
 }
 
-function Star(props) {
+interface StarProp
+  extends Omit<
+    HTMLAttributes<HTMLSpanElement & HTMLButtonElement>,
```

Comment:
I love this simplicity.  But I wonder if it's sufficient.  Because as written this means that every Star instance can have the attributes of spans AND buttons, allowing something like:

```
<Star interactive={false} disabled name="star" value="3" />
```

*Practically*, however, this is probably ok, because *we* control all usage of Star, because it's not a public interface.  For that reason, I could go either way -- keep as-is or make polymorphic.

Reply:
will leave as is